code-challenges.js

```diff
@@ -49,6 +153,38 @@ var stringWithVowels2 = "ILoveJavaScript"
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 // Hint: There is an operator in JavaScript called typeof :)
 
-var refactorTester1 = true
-var refactorTester2 = 42
-var refactorTester3 = "IAmACodingMaster"
+//This seconf part of the function I need to check if any of these are typed, then I need to return the messege 
+
+// var refactorTester1 = true
+// var refactorTester2 = 42
+// var refactorTester3 = "IAmACodingMaster"
+
+
+// const takeOutVowels = (strings) =>{
+
+//     let myNewString = ""
+//     for(let i = 0; i<strings.length; i++){
+//     //  console.log(strings[i])
+//     //If aeiou take it out
+//     let lowerCase = strings[i].toLowerCase()
+//     if(lowerCase !== "a" && lowerCase  !== "e"  && lowerCase  !== "i"  && strings[i] !== "o"  && strings[i] !== "u"){
+//         myNewString+=strings[i]
+//     }
+//      else if(lowerCase === refactorTester1){
+//         console.log("true is not a string")
+//      }
+    
+//      else if(lowerCase === refactorTester2){
+//         console.log("42 is not a string" )
+//      }
+    
+//      else if(lowerCase === refactorTester3){
+//         console.log("mCdngMstr")
+//      }
+//    console.log(refactorTester3)
+//   }
+// return  myNewString
+// }
+// console.log(takeOutVowels(stringWithVowels1))
+// console.log(takeOutVowels(stringWithVowels2 ))
+
```

Comment:
The issue I see here is that you are using variables from outside of the function to make the comparison inside the function. The goal of a function is to be generic and reusable.